### PR TITLE
Add one-stop-shop "make master" target

### PR DESCRIPTION
**Pull request by neiljerram**
(Please see the README.md here for the point of this change.)

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -74,3 +74,9 @@ NETWORKING_CALICO_CHECKOUT?=master
 networking-calico:
 	git clone $(NETWORKING_CALICO_REPO)
 	cd networking-calico && git checkout $(NETWORKING_CALICO_CHECKOUT)
+
+.PHONY: master calico-%
+master:
+	REPO_NAME=$@ utils/create-update-packages.sh
+calico-%:
```
Comment (fasaxc) [inline Makefile]: Worth adding a comment here to say what the `%` is supposed to be.

File: Makefile [hunk 2]
```diff
@@ -74,3 +74,9 @@ NETWORKING_CALICO_CHECKOUT?=master
 networking-calico:
 	git clone $(NETWORKING_CALICO_REPO)
 	cd networking-calico && git checkout $(NETWORKING_CALICO_CHECKOUT)
+
+.PHONY: master calico-%
+master:
```
Comment (fasaxc) [inline Makefile]: How about `publish-master`, just to emphasise that this is going to push something to a public repo?

Comment (neiljerram) [reply Makefile]: In parallel with your review, I wondered about changing this to `make release VERSION=vX.Y.Z`, for compatibility with our component Makefiles?  I think that might address both of your comments here.  WDYT?

Comment (fasaxc) [reply Makefile]: It's `release-publish` in the other makefiles.  `make release` only builds the release

Comment (neiljerram) [reply Makefile]: Cool, I'll do release-publish...

**Pull request merged**